ui/clubhouse.js

```diff
@@ -1266,6 +1267,44 @@ function showOverview() {
         mapNotifyWindow(notify, notify.get_compositor_private());
 }
 
+function onOnboardingHLChanged(isOnboardingHL) {
+    const app = Utils.getClubhouseApp();
+    const notify = app.get_windows().find(w => w.get_role() === 'clubhouse-msg-notify');
+
+    if (!isOnboardingHL && !Main.overview.visible) {
+        _destroyNotifyClone();
+        return;
+    }
+
+    if (notify)
+        mapNotifyWindow(notify, notify.get_compositor_private(), isOnboardingHL);
+}
+
+async function connectOnboarding(cancellable) {
+    try {
+        ONBOARDING_PROXY = await Gio.DBusProxy.new(
+            Gio.DBus.session,
+            Gio.DBusProxyFlags.DO_NOT_AUTO_START,
+            null,
+            'com.endlessm.onboarding',
+            '/com/endlessm/onboarding',
+            'com.endlessm.onboarding',
+            cancellable);
+    } catch (e) {
+        log('error creating onboarding proxy: %s'.format(e.message));
```

Comment:
I think you should check if error=CANCELLED here also to avoid a warning when we cancel on purpose (on disable), but not a blocker.

Reply:
Done